index.js

```diff
@@ -1,27 +1,60 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 // Import of the model Recipe from './models/Recipe.model.js'
-const Recipe = require('./models/Recipe.model');
+const Recipe = require("./models/Recipe.model");
 // Import of the data from './data.json'
-const data = require('./data');
+const data = require("./data");
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+const MONGODB_URI = "mongodb://localhost:27017/recipe-app";
 
 // Connection to the database "recipe-app"
 mongoose
   .connect(MONGODB_URI, {
     useCreateIndex: true,
     useNewUrlParser: true,
-    useUnifiedTopology: true
+    useUnifiedTopology: true,
   })
-  .then(self => {
+  .then((self) => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
+
+  .then(async () => {
+    //Iteration 2: add my recipe
+    await Recipe.create({
+      title: "Sugo al basilico",
+      level: "Easy Peasy",
+      cuisine: "Italian",
+      ingredients: "tomatoes, basil, salt, sugar",
+      creator: "Debora",
+    });
+
+    await Recipe.create(data) //Iteration 3: add recipes from data.json
+      // .then((newRecipes) =>
+      //   newRecipes.forEach((recipe) =>
+      //     console.log(`New receipe added: ${recipe.title}`)
+      //   )
+      // )
+      .catch((err) =>
+        console.log(`Failed adding receipes from data. Err: ${err}`)
+      );
+
+    await Recipe.findOneAndUpdate(
+      //Iteration 4: update value duration to 100
+      { title: "Rigatoni alla Genovese" },
+      { $set: { duration: 100 } },
```

Comment:
Your findOneAndUpdate already knows that the second key is the one to update once it recognises it, so your $set is unnecessary here ($set usually adds new fields to a document; it can also update, but it is used more with the method .aggregate())